### ISSUE TITLE
Please tell me about fedprox and feddyn

### ISSUE THREAD

**Issue by yizhixiaofeiyang** [woodenchild95/FL-Simulator#16]
![image](https://github.com/user-attachments/assets/d07bf7c9-b85a-41b8-b028-6c67f630a76a)
Excuse me, I used FL simulator to train fedprox and feddyn. On the cifar100 dataset, the accuracy has declined. Why？ thank you.

**Comment by woodenchild95** [comment]
@yizhixiaofeiyang hi, can u provide more details about that? It seems to be caused by aggressive hyperparameter settings. It may be necessary to adjust their selections, like the learning rate or proxy coefficient.

**Comment by yizhixiaofeiyang** [comment]
OK, thank you. I will try to modify the learning rate to 0.01, and try to increase it.
